Source/Bundle+Convenience.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+extension Bundle {
+    class var resources: Bundle {
+        let frameworkBundle = Bundle(for: AlertController.self)
+
+        if let bundleURL = frameworkBundle.url(forResource: "SDCAlertView", withExtension: "bundle"),
+            let bundle = Bundle(url: bundleURL) {
+            return bundle
+        }
```

Comment:
I think we can make this a little shorter:

```swift
let sourceBundle = Bundle(for: AlertController.self)
let bundleURL = sourceBundle.url(forResource: "SDCAlertView", withExtension: "bundle")
return bundleURL.map(Bundle.init(url:)) ?? frameworkBundle
```

Reply:
Return type for `bundleURL.map(Bundle.init(url:)) ?? sourceBundle` is 'Bundle??', so it requires additional unwrapping.

Reply:
Ah oops, that should be a `flatMap` then